src/plugins/intel_gpu/tests/unit/fusions/convolution_fusion_test.cpp

```diff
@@ -3786,7 +3803,7 @@ TEST_P(post_ops_optimizations_onednn_eltw_any_sum_eltw_linear, basic) {
         data("out_hi", get_mem(get_single_element_layout(p), 127)),
         data("eltwise_data", get_mem(get_output_layout(p))),
         convolution("conv_prim", input_info("input"), "weights", "bias", p.groups, p.stride, p.dilation, p.pad, p.pad, format::is_grouped(get_weights_layout(p).format)),
-        activation("activation", input_info("conv_prim"), activation_func::relu_negative_slope),
+        activation("activation", input_info("conv_prim"), activation_func::relu),
```

Comment:
could you explain why you changed the function to `relu` from `relu_negative_slope`?

Reply:
changed activation becuz some cases are (multi batch conv and prelu) condition. After check the test, it changed value of expected_fused count for onednn.